app/views/welcome/_subscribers_form.html.haml

```diff
@@ -0,0 +1,17 @@
+%p.text-center
+  %span#subscriber_new_title_js.title Subscribe to our newsletter
```

Comment:
Classes should be listed before IDs (.title should precede #subscriber_new_title_js)